atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -316,7 +314,11 @@ private State initialState() throws SQLException {
         HikariConfig config = new HikariConfig();
 
         // additional connection properties will go in here
-        Properties props = JdbcConfig.getPropertiesFromDbConfig(dbConfig);
+        Properties props = JdbcConfig.getPropertiesFromDbConfig(
```

Comment:
I did this to maintain package boundaries.  The JdbcConfig doesn't know about ConnectionConfig.  I could start moving things between packages, but that will cleanup harder.  I'm happy to go either way on this.